_im-plugin/ism/policies.md

```diff
@@ -162,6 +163,46 @@ Parameter | Description | Type | Required
 
 For information about setting replicas, see [Primary and replica shards]({{site.url}}{{site.baseurl}}/opensearch#primary-and-replica-shards).
 
+### shrink
+
+Allows you to reduce the number of primary shards in your indexes. With this action, you can specify:
+
+- the number of primary shards that the target index should contain.
+- A max shard size for the primary shards in the target index.
+- Specify a percentage to shrink the number of primary shards in the target index.
```

Comment:
Nit: inconsistent capitalization and the third line reads you can specify: Specify a percentage, should be consistent with the others as 'you can specify: a percentage to...'

Reply:
Agreed. Please capitalize the first word of each bullet and make the suggested change to the third bullet.

Reply:
Just a followup comment, when you look at this rendered, the section above has the intro sentence "With this action, you can specify:"

So then each bullet should just start capitalized:
The number of primary. . .
A max shard size. . .
A percentage. . . 

```suggestion
- The number of primary shards that the target index should contain.
- A max shard size for the primary shards in the target index.
- A percentage to shrink the number of primary shards in the target index.
```